test/regression/riscv/RISCVRegressionTest.hh

```diff
@@ -4,7 +4,7 @@
 #include "simeng/arch/riscv/Architecture.hh"
 #include "simeng/arch/riscv/Instruction.hh"
 
-static const char* RISCV_ADDITIONAL_CONFIG = R"YAML(
+static constexpr const char* RISCV_ADDITIONAL_CONFIG = R"YAML(
```

Comment:
Motivation for `constexpr` use? Not sure what compile-time advantage we can get here. Also, do you need the `const` specifier with a `constexpr` specifier?

Reply:
Was playing around with quashing warnings with @tom91136 yesterday. Shouldn't have been added to this PR so will remove